bridge_adaptivity/bridge_lti/provider.py

```diff
@@ -151,14 +162,14 @@ def learner_flow(request, lti_consumer, tool_provider, collection_slug=None, gro
 
     if created:
         log.debug("Sequence {} was created".format(sequence))
-        start_activity = module_utils.choose_activity(sequence_item=None, sequence=sequence)
+        start_activity = module_utils.choose_activity(sequence=sequence)
         if not start_activity:
             log.warning('Instructor configured empty Collection.')
             return stub_page(
                 request,
                 title="Warning",
                 message="Cannot get the first question to start.",
-                tip="Please try again later"
```

Comment:
Why do you add changes to the part of the function which is not changed in your PR?